src/python/espressomd/integrate.pyx

```diff
@@ -440,7 +440,7 @@ IF STOKESIAN_DYNAMICS:
             return True
 
         def _set_params_in_es_core(self):
-            integrate_set_bd()
+            integrate_set_sd()
```

Comment:
Looks like I messed up the merge big time here. Troubleshooting the consequences of that typo and the segfault in the subsequent merge (which was dropped from this PR) cost me and Carl a lot of man hours and frustration. TDD was introduced in our team to avoid this situation from happening.